### ISSUE TITLE
Implement filtering of packages, services and functions by type

### ISSUE THREAD

**Issue by jbonnet** [sonata-nfv/son-gkeeper#671]
Once users are validated, we should provide consumers of this information with a richer data about packages, services and functions, namely:
* those that are **public**;
* those the user is the **owner**;
* those the user has **bought a licence**.

The GK can return a JSON like in (example for packages, but it will be similar for services and functions):
```
{ 'public-packages' : [...], 'owned-packages' : [...], 'licenced-packages' : [...]}
```
@srodriguezOPT, @mzacharioudakis, this was what we've talked about in one of the Paris meeting coffee breaks. Do you agree with this suggestion? When do you think we could implement this?

**Comment by jbonnet** [comment]
Hi, @srodriguezOPT, @mzacharioudakis 
Have you had the chance to look at this?

**Comment by srodriguezOPT** [comment]
hi @jbonnet , @mzacharioudakis 

I've realised right now. 

Are we talking about something like this?
GET /api/v2/packages/:license_type (public/private)
GET /api/v2/packages/:owner
GET /api/v2/services/:license_type (public/private)
GET /api/v2/services/:owner
GET /api/v2/functions/:license_type (public/private)
GET /api/v2/functions/:owner
GET /api/v2/licenses/services/:user_id
GET /api/v2/licenses/functions/:user_id

**Comment by jbonnet** [comment]
@srodriguezOPT 
So, what we have agreed by email: we'll have 2 lists of services,

1. ‘instantiatable’ (`public` or with licence `acquired` or `owned`)
1. ‘must buy’

**Comment by jbonnet** [comment]
Closed by PR #748
